lib/publiccloud/instance.pm

```diff
@@ -85,6 +85,66 @@ sub run_ssh_command {
     }
 }
 
+
+=head2 ssh_script_run
+
+    ssh_script_run(command[, timeout => 90][, ssh_opts =>'..'][, username => 'XXX'][, no_quote => 0]);
+
+Runs a C<command> via ssh in the given PC instance and returns the return code
+Timeout can be set by C<timeout> or 90 sec by default.
+By default, the command is passed in single quotes to SSH.
+To avoid quoting use C<<no_quote=>1>>.
+With C<<ssh_opts=>'...'>> you can overwrite all default ops which are in
+C<<$instance->ssh_opts>>.
+Use argument C<username> to specify a different username then
+C<<$instance->username()>>.
+
+This is a wrapper function for run_ssh_command
```

Comment:
If it is a wrappe, I woudn't copy the perlpod of the original function. Just pointing to it and describe the differences.

Reply:
I've removed the wrapper function and refactored the original method instead.